Main Model/firm-procedures.nls

```diff
@@ -415,9 +353,12 @@ to buy-input-goods
     let firms-to-try framework-agreement-neighbors with [firm-type = i]
     while [remaining-demand >= (0.05 * daily-input-demand i) and any? firms-to-try][
```

Comment:
daily-input-demand isn't a good name now that we are only doing stuff once per month (or transactions-per-month times per month).